airflow/README.md

```diff
@@ -1,32 +1,32 @@
+The following folder contains the project level directory for all our [Apache Airflow](https://airflow.apache.org/) [DAGs](https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/dags.html). Airflow is an orchestration tool that we use to manage our raw data ingest. Airflow DAG tasks are scheduled at regular intervals to perform data processing steps, like unzipping raw GTFS zipfiles and writing the contents out to Google Cloud Storage.
 
-Our DAGs are deployed automatically to Google Cloud Composer when a PR is merged into the `main` branch; see the section on deployment below for more information.
+Our DAGs, plugins, and changes to [the Airflow image we use for local testing](./Dockerfile) are deployed automatically to Google Cloud Composer when a PR is merged into the `main` branch; see the section on deployment below for more information. Images used by some of our Airflow DAGs (i.e. those which are configured with PodOperators) are deployed separately, often also automatically upon merged changes to the relevant Dockerfiles in the repository. System configuration for worker count, environment variables, and overrides of Airflow configs are deployed via the [Composer web console](https://console.cloud.google.com/composer/environments?project=cal-itp-data-infra), not via an automated process, but additional dependencies that we add to the standard Composer-managed Airflow install (listed in [requirements.txt](./requirements.txt)) _are_ deployed automatically upon merged changes.
 
 ## Structure
 
 The DAGs for this project are stored and version controlled in the `dags` folder. Each DAG has its own `README` with further information about its specific purpose and considerations. We use [gusty](https://github.com/pipeline-tools/gusty) to simplify DAG management.
 
 Each DAG folder contains a [`METADATA.yml` file](https://github.com/pipeline-tools/gusty#metadata) that contains overall DAG settings, including the DAG's schedule (if any).
 
-The logs are stored locally in the `logs` folder. You should be unable to add files here but it is gitkeep'ed so that it is avaliable when testing and debugging.
+The logs are stored locally in the `logs` folder. You should be unable to add files here but the folder utilizes [a .gitkeep file](https://stackoverflow.com/a/7229996) so that it is consistently avaliable when testing and debugging.
 
 Finally, Airflow plugins can be found in `plugins`; this includes general utility functions as well as custom [operator](https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/operators.html) definitions.
 
-## Developing Locally
+## Testing Changes
```

Comment:
Did not retain it in the head, since similar headers in other READMEs involve non-local steps for testing, but I added explanatory text making clear the local state of Airflow in testing in [a503dd4](https://github.com/cal-itp/data-infra/pull/3156/commits/a503dd496b546a9fd8a0f3f4872b8d1b8bf8be16)